dashboard/src/components/ConfigLoader/ConfigLoader.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from "react";
 
 import UnexpectedErrorPage from "../../components/ErrorAlert/UnexpectedErrorAlert";
-import LoadingWrapper, { ILoadingWrapperProps } from "../../components/LoadingWrapper";
+import LoadingWrapper from "../../components/LoadingWrapper";
+import { ILoadingWrapperProps } from "../LoadingWrapper/LoadingWrapper";
```

Comment:
These look like relative paths to the same component?

Reply:
nope, (notice the inner LoadingWrapper) but it could be